bears/general/InvalidLinkBear.py

```diff
@@ -80,19 +81,25 @@ def find_links_in_file(file, network_timeout, link_ignore_regex,
                                         # Unbalanced parenthesis
             (?<!\.)(?<!,)               # Exclude trailing `.` or `,` from URL
             """, re.VERBOSE)
+        xml_namespace_links = set()
         for line_number, line in enumerate(file):
             match = regex.search(line)
             if match:
                 link = match.group()
+                xmlns_match = xmlns_regex.search(line)
+                if xmlns_match and link in xmlns_match.groups():
+                    xml_namespace_links.add(link)
+                    yield line_number + 1, link, '', True
                 if not (link_ignore_regex.search(link) or
-                        fnmatch(link, link_ignore_list)):
+                        fnmatch(link, link_ignore_list) or
+                        link in xml_namespace_links):
```

Comment:
argh. This should be happening already for all links. See https://github.com/coala/coala-bears/issues/1319

We should actually report additional occurrences, but re-use the initial context of the url so that extra occurences are a lower severity.